+dj/kill_quick.m

```diff
@@ -25,11 +25,16 @@
         qstr = sprintf('%s AND (%s)', qstr, restriction);
     end
 
-    sprintf('%s', qstr)
-
     res = query(connection, qstr);
+
+    if isfield(res, 'ID')  % MySQL >8.x variation
+        id_field = 'ID';
+    else
+        id_field = 'id';
+    end
```

Comment:
Unfortunately, this won't work either. MySQL is not consistent on column case across `information_schema`... For this table specifically, it is upper-cased across all major releases. I think it is best to explicitly lower all column names, then index (the pattern that is emerging on `datajoint-python` side). It will be safe across MySQL changes in the future as well.

Reply:
if it is upper cased across major releases why are we changing it? i thought that was the original problem...

and either way, why wouldnt this work for this case? we only care about accessing ID in this block..
... so seems better/simpler than something like:

```
res = cell2struct(struct2cell(res), ... % downcase columns for MySQL >8x
                  cellfun(@lower, fieldnames(res), ...
                          'UniformOutput', false));
```

so correspondingly, after thinking probably simpler to just do the ID field only on python side e.g.:

```
dct1 = {'a': 1}
dct2 = {'A': 1}
assert dct1['a' if 'a' in dct1 else 'A'] == 1
assert dct2['a' if 'a' in dct2 else 'A'] == 1
```

Reply:
In my previous [PR in datajoint-python](https://github.com/datajoint/datajoint-python/pull/819), we discussed this very topic. Originally, I had done it as you suggest, however, this meant that for some tables (i.e. `key_column_usage`) I was explicitly lower-casing while for `processlist`, I left it all as upper-case (since wasn't required). When discussing with the team, it was determined (you also agreed to this) that it would be best to lower-case all results to preserve consistency across the board and ensure safety across MySQL updates (`key_column_usage` was one that broke on a minor release, `8.X`). I am merely proposing here that we keep the same approach as was done in `datajoint-python` or determine an approach that is consistent.

That said, I do loathe that the syntax in MATLAB can be more verbose generally, but I would have done it like this:
```matlab
res = cell2struct(struct2cell(res), lower(fieldnames(res)));
```

On python, side: we only need access to `id` on `dj.kill_quick`, however, on `dj.kill` we access all of the columns. Perhapds, there might be a better way to display it to avoid indexing (similarly to how we are simply displaying the query result from `mym` in `datajoint-maltlab`).

Reply:
sorry yes, knew we had come to the agreement for some reason & thought it was fully discussed, but got confused here based on the 'not work' comment.

for kill using all fields - matlab is a bit different than python since we're just relying on 'raw output' in the 
display part (line 54), that said, the better non-cellfun version seems clean enough to 
remove my 'id only' if/else, will adjust.